src/components/TodoList.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+ 
+    
+            <!-- <div >
+                
+            <button class="btn" v-for="post in posts" v-bind:title="post.data" v-bind:key="post.id" 
+            v-bind:style = '{"text-decoration" : (post.isCompleted? "line-through" : "none" )}' 
+            @click="post.isCompleted = !post.isCompleted">
+                
+                {{ post.data }}
+                
+               
+            </button>
+             </div> -->
+            
+       
+
+        
+        <div class="div">
+            
+            <div class="inp">
+                <input type="text" v-model="posts.data" placeholder="Type in something">
+                <button  @click="addItem(posts.data)">Add Item</button>
+                <button @click="clearAll(posts.id)">Clear All</button>
+            </div>
+                
+                <div class="submit" >
+
+                    <h1
+            v-for="(post,k) in posts" v-bind:title="post.data" v-bind:key="k" 
+            v-bind:style = '{"text-decoration" : (post.isCompleted? "line-through" : "none" )}' 
```

Comment:
prefer putting this style in a class and conditionally adding the class instead of the style